js/ui/text_box/ui.text_editor.base.js

```diff
@@ -710,7 +710,7 @@ const TextEditorBase = Editor.inherit({
             case 'text':
                 break;
             case 'value':
-                this._updateValue();
+                this._updateValue(this._useValuesToUpdate);
```

Comment:
Why do we need this?

```suggestion
                this._updateValue();
```